[Refact] #14 memo 컴포넌트화

sewonkimm opened the pull request.

sewonkimm committed:
[Refact] #14 memo 컴포넌트화

JaeHyeonKim19 commented on app/src/components/Memo/styles.scss:
```diff
@@ -0,0 +1,26 @@
+  display: flex;
+  flex-direction: column;
+
+  width: calc(100% - 2rem);
```
음 어떤 배경으로 다음과 같이 width를 지정하셨는지 알 수 있을까요?

sewonkimm replied:
memo_list_container에서 align-itmes 속성을 flex-start로 설정한 채로 memo div를 가운데 정렬하려고 저렇게 작성했습니다. memo_list_container align-itmes 속성을 center로 변경하는게 좋을 것 같네요!

JaeHyeonKim19 commented on app/src/containers/MemoListPage/index.tsx:
```diff
@@ -10,20 +11,9 @@ const MemoListPage = () => {
       <MemoSetting />
 
       <div className={styles.memo_list_container}>
```
`<section />`이라는 시멘틱 태그를 활용하면 의미적으로 더 좋을 것 같아요.

JaeHyeonKim19 reviewed (commented):
수고하셨습니다. 궁금한 점 있어서 코멘트 남겼어요.

sewonkimm committed:
[Refact] #14 css 조정

- 가독성을 위해 align-itmes를 flex-start에서 center로 수정하고, width의 rem 단위 제거
- <section> 태그 사용

soyoungjeong reviewed (approved):
수고하셨습니다 ;)

sewonkimm committed:
Merge branch 'develop' of https://github.com/mojimojicorp/MojimojiMemoji into refact/#14
